package.json

```diff
@@ -10,8 +10,8 @@
     "start": "cross-env node server/index.js",
     "generate": "nuxt generate",
     "lint": "eslint --ext .js,.vue .",
-    "styleguide": "cd ./styleguide && yarn dev",
-    "styleguide:build": "cd ./styleguide && yarn build:lib && cd ../",
+    "styleguide": "echo 'Command styleguide is Deprecated!'",
```

Comment:
Why not simply remove? Do we have dependencies in our CI/CD pipeline that would break?

Reply:
I was thinking about user that still typing that commands ^^